config/builds.js

```diff
@@ -37,7 +37,9 @@ const builds = buildConfigs
   })
   .map(buildConfig => {
     const name = buildConfig.name || '';
-    const env = buildConfig.env || {};
+    const env = buildConfig.env || {
+      SKU_TENANT: args.tenant || ''
```

Comment:
Yeah, I had a comment on that but it got blown away.

Could just spread it in?  We could have a convention of not specifying your own SKU_* in build configs and then supplying all of them every time.